gtk4/Gir.toml

```diff
@@ -1657,6 +1657,11 @@ name = "Gtk.PasswordEntry"
 status = "generate"
 trust_return_value_nullability = false
 
+[[object]]
+name = "Gtk.PasswordEntryBuffer"
+status = "generate"
+version = "4.4"
```

Comment:
I suppose this was upstreamed/reported?

Reply:
ah yeah, it was not introspected correctly: https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/3747